### DISCUSSION TITLE
点歌：沐神代表作：参数服务器 Scaling Distributed Machine Learning with the Parameter Server

### DISCUSSION THREAD

**Discussion by cnjackhu** [mli/paper-reading#147] [Ideas]
沐神你好，能不能带我们重读沐神经典代表作，parameter server,非常感谢！

**Comment by Sanzo00** [comment]
附议，想听听论文背后的故事。

**Reply by mli** [reply]
讲讲背后的故事这个想法很不错啊，我考虑下。之前我没考虑讲这一篇是因为研究方向里人不多

**Reply by BIGBALLON** [reply]
表示很感兴趣！！！！

**Reply by LuYang-2023** [reply]
啊啊啊，听故事，附议附议

**Reply by Guohui-Peng** [reply]
> 啊啊啊，听故事，附议附议

已经讲过了。
https://www.bilibili.com/video/BV1YA4y197G8/?spm_id_from=333.999.0.0

**Reply by LuYang-2023** [reply]
> > 啊啊啊，听故事，附议附议
> 
> 已经讲过了。 https://www.bilibili.com/video/BV1YA4y197G8/?spm_id_from=333.999.0.0

抱歉抱歉，是我草率了，没看见，我还以为纯故事嘞，哈哈哈
